v3/packages.json

```
@@ -3068,6 +3068,19 @@
         { "filename": "plugins/lwmuxer.auf" }
       ]
     },
+    {
+      "id": "kame/pointlinetext",
+      "name": "丸付き折れ線テキスト",
+      "overview": "丸付折線テキスト カスタムオブジェクトスクリプト",
+      "description": "丸付き引き出し線とテキストがセットになったカスタムオブジェクトのスクリプトです。",
+      "developer": "カメ",
+      "pageURL": "https://www.nicovideo.jp/watch/sm22558183",
+      "downloadURLs": [
+        "https://takeshima.halfmoon.jp/room/aviutl/#pointlinetext_obj"
+      ],
+      "latestVersion": "ver.001",
+      "files": [{ "filename": "script/丸付折線テキスト.obj" }]
```

これまでのところ、検索性を考えてスクリプトをフォルダ分けするようにしています。
```suggestion
      "files": [{ "filename": "script/kame/丸付折線テキスト.obj" }]
```